pkg/stack/unwind/maps.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package unwind
+
+import (
+	"fmt"
+
+	"github.com/prometheus/procfs"
+)
+
+// ExecutableMapping represents an executable memory mapping.
+type ExecutableMapping struct {
+	LoadAddr   uint64
+	StartAddr  uint64
+	EndAddr    uint64
+	Executable string
+	mainExec   bool
+}
+
+// MainObject returns whether this executable is the "main executable".
+// which triggered the loading of all the other mappings.
+//
+// We care about this because if Linux ASLR is enabled, we have to
+// modify the loaded addresses for the main object.
+func (pm *ExecutableMapping) MainObject() bool {
```

Comment:
```suggestion
func (pm *ExecutableMapping) IsMainObject() bool {
```
Just for following the existing method naming conventions

Reply:
Good call!